redbpf/src/lib.rs

```diff
@@ -1893,6 +1902,19 @@ impl Map {
                         }
                     }
                 }
+            } else if let Some(ENOTSUPP) = io::Error::last_os_error().raw_os_error() {
+                debug!(
+                    "Since kernel does not support BTF for {}, BTF is ignored when creating the map `{}'", map_type_name(config.type_), name
```

Comment:
"The kernel does not support BTF for {}. Ignoring BTF for map `{}`." Would be better style.